src/controllers/auth/login.ts

```diff
@@ -1,10 +1,60 @@
-import { Express } from '../../types';
-import { isTest } from '../../config';
+import { Response } from 'express';
+import jwt from 'jsonwebtoken';
+import bcrypt from 'bcrypt';
+import { Developer as DeveloperT, Express } from '../../types';
```

Comment:
`DeveloperT` isn't a clear name

```suggestion
import { Developer as DeveloperType, Express } from '../../types';
```